tests/common/fixtures/duthost_utils.py

```diff
@@ -754,7 +754,7 @@ def convert_and_restore_config_db_to_ipv6_only(duthosts):
         if config_db_modified[duthost.hostname]:
             logger.info(f"config changed. Doing config reload for {duthost.hostname}")
             try:
-                config_reload(duthost, wait=120)
+                config_reload(duthost, wait=120, wait_for_bgp=True)
```

Comment:
I have a concern that, after the config_reload executed, the original session with based on the IPv4 will be gone,
so the duthost is not usable any more, at least during the development it was failing.

That's why we only execute a simple wait=120 rather than safe_reload=True.

And the line 763, duthosts.reset() is to recreate  the sessions with IPv6.

From my view, a safer way is to wait for bgp up on line 769.

But we can adapt Bing's change for now, now that it passed on the physical test.